templates/headers.hbs

```diff
@@ -30,4 +30,7 @@
 <meta name="theme-color" content="#ffffff">
 
  <!-- atom -->
- <link type="application/atom+xml" rel="alternate" href="https://blog.rust-lang.org/{{blog.prefix}}feed.xml" title="{{blog.title}}" />
+<link type="application/atom+xml" rel="alternate" href="https://blog.rust-lang.org/{{blog.prefix}}feed.xml" title="{{blog.title}}" />
+
+ <!-- scripts -->
+<script defer src="{{root}}scripts/highlight.js"></script>
```

Comment:
If I remember correctly, one way to improve website loading times is to put `<script>` at the very end of the HTML, so I think this change here could theoretically lead to a perf. disadvantage compared to the previous solution

Reply:
Hi @apiraino, you are correct. Adding a script tag at the end of the body will have its own benefits, but it will start to fetch the script after parsing the HTML. However, with the `defer` attribute, we can achieve the same but with parallel fetching during HTML parsing, and it is executed at the end of the body.

Here is a small blog which explains the advantages of it: [Improving Third-Party Web Performance at The Telegraph](https://medium.com/the-telegraph-engineering/improving-third-party-web-performance-at-the-telegraph-a0a1000be5#4123)

Reply:
Google's Article -: https://web.dev/articles/efficiently-load-third-party-javascript#defer